src/main/java/org/example/models/PlayerStatus.java

```diff
@@ -31,8 +35,9 @@ public WeaponModel getWeaponInHand() {
 
     public void setWeaponInHand(WeaponModel weapon, Weapons weaponType) {
```

Comment:
For better readability, I'd change the name to something like 'equipWeapon'. Usually only setters start with 'set'.

To simplify the code, I'd modify the 'name' property to be of type 'Weapons' enum, instead of String and, maybe, rename it to 'weapon' or 'weaponName'. This means you no longer need the second parameter.

Reply:
Solved, albeit the return value is never used, weapon check is already done inside this method.

Reply:
https://github.com/razvanexu/JavaLearn_Player_Status/pull/2/commits/535b2dd436c7f2639806dcf30a990a005da32727